tests/test_cvdupdate.py

```diff
@@ -0,0 +1,67 @@
+import json
+from pathlib import Path
+
+from tests.fixtures.revert import revert_homedir
+import pytest
+
+from cvdupdate.cvdupdate import CVDUpdate
+
+def test_instantiation(revert_homedir):
+    c = CVDUpdate()
+
+def test_alternate_config_locations(revert_homedir, tmp_path):
```

Comment:
Since the tests have the potential to mess up an existing installation, perhaps we should only test with alternate config locations. I would recommend creating a new directory in the temp directory.

Reply:
I tried for a while to make an overlay FS work inside a docker container, but couldn't quite get it.

I thought it would be important to be able to test with the default location since that is the default code path, though admittedly I only added the most basic of all tests for that (line 9).  Still, IMO, that could and should be expanded.  That is why line 16 will abort the test if it finds an existing `~/.cvdudpate/`.  Perhaps I should make that more explicit with some sort of error message?  I could even probably make it impossible to run any of these tests with an existing `~/.cvdupdate/` via some session fixture in pytest?